src/lib/SignatureHelper.ts

```diff
@@ -141,3 +177,67 @@ export function signatureToVRS(
 
   return { v, r, s };
 }
+
+export function signatureToSolidityStruct(
+  typedSignature: TypedSignature,
+): {
+  vType: string,
+  r: string;
+  s: string
+} {
+  const rawSignature = typedSignature.slice(0, 132);
+  const { v, r, s } = signatureToVRS(rawSignature);
+  return {
+    vType: `0x${v}01`, // TODO
```

Comment:
@BrendanChou Can I hard code the type as `1` like this?

Reply:
I think the type `01` would come from `typedSignature.slice(132, 2)`